README.md

```diff
@@ -208,6 +208,33 @@ oktaOidc.signOutOfOkta(authStateManager, from: self) { error in
 ```
 Sample app [example](https://github.com/okta/samples-ios/blob/master/browser-sign-in/OktaBrowserSignIn/SignInViewController.swift#L62-L74)
 
+### signOut
+
+This method helps to perform mutli-step sign out flow. Provide options that you want to perform and sdk will run them as a batch. Available options are:
```

Comment:
This method helps to perform a multi-step sign-out flow. The method provides options that you want to perform and the SDKl runs the options as a batch. The available options are: